package.json

```diff
@@ -60,8 +60,8 @@
     "negotiator": "^0.6.0",
     "object-values": "^1.0.0",
     "parseurl": "^1.3.0",
-    "raml-sanitize": "^1.1.2",
-    "raml-validate": "^1.0.6",
+    "raml-sanitize": "jstoiko/node-raml-sanitize#raml1.0",
+    "raml-validate": "jstoiko/node-raml-validate#raml1.0",
```

Comment:
Remember to update to 1.1.0 according to https://github.com/mulesoft/node-raml-validate/pull/5/files